src/components/pages/donation_form/subpages/AddressPage.vue

```diff
@@ -17,32 +17,35 @@
 		</form>
 
 		<form id="address-type-selection" @submit="evt => evt.preventDefault()">
-			<feature-toggle>
-				<address-type-basic
-					slot="campaigns.address_type_steps.direct"
-					v-on:address-type="setAddressType( $event )"
-					v-on:set-full-selected="setFullSelected"
-					:disabledAddressTypes="disabledAddressTypes"
-					:is-direct-debit="isDirectDebit"
-					:initial-address-type="addressTypeName"
-				/>
-				<address-type-all-options
-					slot="campaigns.address_type_steps.preselect"
-					v-on:address-type="setAddressType( $event )"
-					v-on:set-full-selected="setFullSelected"
-					:disabledAddressTypes="disabledAddressTypes"
-					:is-direct-debit="isDirectDebit"
-					:initial-address-type="addressTypeName"
-				/>
-				<address-type-full-or-email
-					slot="campaigns.address_type_steps.full_or_email"
-					v-on:address-type="setAddressType( $event )"
-					v-on:set-full-selected="setFullSelected"
-					:disabledAddressTypes="disabledAddressTypes"
-					:is-direct-debit="isDirectDebit"
-					:initial-address-type="addressTypeName"
-				/>
-			</feature-toggle>
+			<FeatureToggle>
+				<template #campaigns.address_type_steps.direct>
+					<address-type-basic
+						v-on:address-type="setAddressType( $event )"
+						v-on:set-full-selected="setFullSelected"
+						:disabledAddressTypes="disabledAddressTypes"
+						:is-direct-debit="isDirectDebit"
+						:initial-address-type="addressTypeName"
+					/>
+				</template>
+				<template #campaigns.address_type_steps.preselect>
+					<address-type-all-options
+						v-on:address-type="setAddressType( $event )"
+						v-on:set-full-selected="setFullSelected"
+						:disabledAddressTypes="disabledAddressTypes"
+						:is-direct-debit="isDirectDebit"
+						:initial-address-type="addressTypeName"
+					/>
+				</template>
+				<template #campaigns.address_type_steps.full_or_email>
+					<address-type-full-or-email
+						v-on:address-type="setAddressType( $event )"
```

Comment:
I assume that in Vue3 `slot` attributes are deprecated, so we use this other template syntax from now on?
In that case, please also update the syntax in the `HOWTO_Create_an_a_b_test.md` documentation in the `fundraising-application` repo.